content/en/news/releases/1.7.x/announcing-1.7.1/index.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Announcing Istio 1.7.1
+linktitle: 1.7.1
+subtitle: Patch Release
+description: Istio 1.7.1 patch release.
+publishdate: 2020-09-10
+release: 1.7.1
+aliases:
+    - /news/announcing-1.7.1
+---
+
+This release contains bug fixes to improve robustness. This release note describes what’s different between Istio 1.7.0 and Istio 1.7.1
+
+{{< relnote >}}
+
+# Changes
+
+
+
+- **Added** Envoy [ext authz and gRPC access log API support](https://github.com/istio/istio/wiki/Enabling-Envoy-Authorization-Service-and-gRPC-Access-Log-Service-With-Mixer) in Mixer,
```

Comment:
You shouldn't use authz here, but expand it out and remove it from .spelling. If you want to use authz, you can surround it with single quotes to get around the spellcheck.

Reply:
I added single quotes to get around spellcheck. Thanks.